### PR TITLE
Task/contributor guidelines

### PR DESCRIPTION
**Pull request by filbabic**
Contribution guidelines, doesn't affect the codebase.

### CODE REVIEW

**Review by skydoves — APPROVED** [review]
Overall looks good to me! 👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+### Important modules & files 🧱
+
+- `stream-chat-android-client`: Holds all the code connected to the **low-level-client**, used to request data from and send information to the API.
+- `stream-chat-android-offline`: As an addition to the **client** module, you can use **offline** to introduce persistence and caching to your app.
+- `stream-chat-android-ui-common`: Common UI-related code and utility functions.
+
+- `stream-chat-android-compose-sample`: The main sample app featuring the Compose SDK. 
+- `stream-chat-android-compose`: Our Compose SDK that features channels and messages components, as well as other utilities.
+- `stream-chat-android-ui-components-sample`: The main sample app featuring the XML (UI Components) SDK. 
+- `stream-chat-android-ui-components`: Our XML (UI Components) SDK that features channels and messages components, as well as other utilities.
```
Comment (skydoves) [inline CONTRIBUTING.md:34]: Let me make a modularization graph like this when I have a moment. 😄  https://github.com/GetStream/AvengersChat/blob/main/figure/figure4.png

Comment (adasiewiczr) [reply CONTRIBUTING.md:34]: Nice! We should also include `stream-chat-android-state`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+We'll guide you through our repository structure, present our styling guidelines and describe the pull request and approval process.
+
+These are guidelines and don't have to be followed to a letter, if you're looking to contribute, feel free to do your best and what you're comfortable with, and we'll take care of the rest! 🤗
+
+---
+
+# If I have a question, do I need to read this guide? 💬
+
+Not really! This guide is focused on contributions, so if you have questions, want to report a bug or want to request a new feature, use our [Android Chat SDK Task and Bug Report](https://github.com/GetStream/stream-chat-android/issues/new/choose) templates.
```
Comment (adasiewiczr) [inline CONTRIBUTING.md:11]: Should we redirect to discussions instead or at least mention them?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+* Proposing feature requests and SDK improvements.
+* Creating pull requests to improve our codebase.
+
+Let's go over the process for each of these items!
+
+## Opening issues & bug reports 🐛
+
+Using the GitHub repository and project management system, you can open issues in any public repo. Before filing bugs, take a look at our existing backlog. For common items or reports, there might be an existing ticket on GitHub.
+
+To quickly narrow down the number of tickets on Github, try filtering based on the label that best suites the bug or a part of our SDK.
```
Comment (adasiewiczr) [inline CONTRIBUTING.md:63]: Should we also mention our projects? Maybe sometimes it might be easier to find issues there?

**Review by adasiewiczr — COMMENTED** [review]
Looks amazing! Should we also mention this doc in the README?

File: CONTRIBUTING.md (hunk 2 above)
Comment (filbabic) [reply CONTRIBUTING.md:11]: That's a good point - I'll add the first step to be look into discussions

**Review by adasiewiczr — APPROVED** [review]
Amazing job!

File: CONTRIBUTING.md (hunk 1 above)
Comment (filbabic) [reply CONTRIBUTING.md]: @skydoves did you perhaps have any time for the diagram? If not, we could just merge this for now and then add the diagram at some point in the future

Comment (skydoves) [reply CONTRIBUTING.md]: Let's merge this PR first. I will make a new PR for the diagram in the near future.

**Pull request merged**